spec/person_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classes/person'
+
+describe Person do
+  age = 22
+  name = 'Akuma'
+  let(:person) { Person.new age, name }
+
```

Comment:
This milestone requires that there are unit tests for all the public methods in all the classes.

- There are three methods in the `Person` class that are not tested. Kindly make sure that the `can_use_services?`,  `validate_name`, and `add_rental` methods are tested for the `Person` class.

Reply:
ok thank you I will add those soon